pkg/jwt/handler.go

```diff
@@ -41,9 +42,10 @@ func (j *Handler) Login(config config.Credentials) http.HandlerFunc {
 		factory := provider.Factory{}
 		p := factory.Build(r.URL.Query().Get("provider"), config)
 
-		span = opentracing.FromContext(r.Context(), "token.verify")
+		_, span = trace.StartSpan(r.Context(), "token.verify")
```

Comment:
same as command above

Reply:
just checked the code, here if can pass oc transport to github client, this way we can have traces and metrics for http calls

Reply:
however I think it's too much for this PR, we can manage to improve later if you think it's take too much

Reply:
apparently not too hard, we just need to inject the transport to GH client as follows:

```go
c := &http.Client{
  Transport: &ochttp.Transport{}
}

client := github.NewClient(c)
```

Reply:
`ochttp` is now used to collect http client stats and traces here:
https://github.com/hellofresh/janus/blob/ed29c5817f408b86a17e95b526d4baae57ff3c59/pkg/proxy/register.go#L62-L69